src/components/Header/Header.tsx

```diff
@@ -23,22 +27,25 @@ export const Header: React.FC = () => {
                 <li>
                   <a
                     href="#home"
-                    className="link nav__link nav__is-active"
+                    className={cn('link', 'nav__link', {
+                      'nav__is-active': activeLink === 'home',
+                    })}
+                    onClick={() => setActiveLink('home')}
                   >
                     Home
                   </a>
                 </li>
```

Comment:
![image](https://github.com/fe-may23-scriptsquad/phone-catalogue-fe/assets/72705169/48c234c5-6461-4ed3-a174-c9a850009b24)

Move this to a separate component, please.
Also, only the Home button can be set to active.